docs/user-guide/v3_migration.rst

```diff
@@ -154,8 +153,30 @@ Common replacements:
    change ensures that all store methods are non-blocking and are as performant as
    possible.
 
-Beyond the changes store interface, a number of deprecated stores were also removed in
-Zarr-Python 3. See :issue:`1274` for more details on the removal of these stores.
+Removed store API
+^^^^^^^^^^^^^^^^^
+
+``zarr.storage.contains_array()`` and ``zarr.storage.contains_group()`` have been removed.
```

Comment:
[`contains_array`](https://github.com/zarr-developers/zarr-python/blob/f9b5a3bf47e981c58a66222a15bd0df9413b8662/src/zarr/storage/_common.py#L437) and [`contains_group`](https://github.com/zarr-developers/zarr-python/blob/f9b5a3bf47e981c58a66222a15bd0df9413b8662/src/zarr/storage/_common.py#L472) have not been removed

Reply:
From a user perspective they have, because they're in a private sub-module. I can make them public in this PR if they were always intended to be public?

Reply:
I think we can say that `contains_array` and `contains_group` are not yet part of the public API. I don't have any problem adding them to the public API but maybe that's better for another PR.